cli_tools/gce_vm_image_import/cli/cli.go

```diff
@@ -0,0 +1,145 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
Why is this refactoring needed for this test? We already test image import wrapper directly in cli_tools_e2e_test/gce_image_import_export/test_suites/import/cli_tests.go where we call the binary itself.

Reply:
By using a compile-time dependency, rather than a snapshot dependency, this test asserts the correctness of the current codebase at a particular commit. That helps when developing locally (I'm testing what I'm working on), and provides flexibility for running this at different stages in a CI system.

Reply:
Our test runners are built at the same snapshot as the code they are testing, so I don't see a benefit for our prod tests. 

I can see a benefit in this refactoring when running locally, because it skips the need for `go build` on wrappers themselves and copy them to the same folder as test runner. On the other hand, these two steps can be easily automated with a simple script that also runs a test.

Reply:
Here are some benefits I see:

- When running locally, you can execute a test in a terminal using `go test`
- In an IDE you can step through the test
- Running locally is the same as prow (they both run `go test`), so you don't have inconsistencies due to local workarounds being different than how the test is executed on the CI system
- When tests pass on testgrid, it's clear what code was being tested (the git commit of the test code is the same as the git commit of the code-under-test)
  - https://github.com/GoogleCloudPlatform/testgrid/blob/master/config.md#column-headers